.github/workflows/golangci-lint.yml

```diff
@@ -17,6 +17,10 @@ jobs:
         working-directory:
         - ""
         - api
+        - hack/tools/release
```

Comment:
This does not get linted using this config.

Reply:
I'm not sure what you mean. The /.golangci-lint.yml config is used when golangci-lint is run.

If meaning when running `make lint`, I completely missed adding it. Thank you for bringing that up. Should be good picked up now.